internal/network/command_network_link_service_delete.go

```diff
@@ -46,13 +45,10 @@ func (c *command) networkLinkServiceDelete(cmd *cobra.Command, args []string) er
 	}
 
 	deleteFunc := func(id string) error {
-		if err := c.V2Client.DeleteNetworkLinkService(environmentId, id); err != nil {
-			return fmt.Errorf(errors.DeleteResourceErrorMsg, resource.NetworkLinkService, id, err)
-		}
-		return nil
+		return c.V2Client.DeleteNetworkLinkService(environmentId, id)
```

Comment:
The `Delete` (and `DeleteWithoutMessage`) functions already wrap these errors w/ a message nearly identical to `DeleteResourceErrorMsg`, so these changes are to remove redundant phrasing from error messages like `Error: failed to delete n-abc123: failed to delete network "n-abc123": ...`